### PR TITLE
docstring indentation changes #2437

### PR DESCRIPTION
**Pull request by AkshJain99**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: bears/python/PEP8Bear.py [hunk 1]
```diff
@@ -28,15 +28,15 @@ def run(self, filename, file,
             local_pep8_config: bool = False,
             ):
         """
-        Detects and fixes PEP8 incompliant code. This bear will not change
+        Detects and fixes PEP8 incompliant code.This bear will not change
```
Comment (apb7) [inline bears/python/PEP8Bear.py]: Please do not remove the space after the full stop, that is, you need not change anything in this line. Thanks!

Comment (AkshJain99) [reply bears/python/PEP8Bear.py]: okay can you tell me what exactly I need to do to solve this issue

File: bears/python/PEP8Bear.py [hunk 2]
```diff
@@ -28,15 +28,15 @@ def run(self, filename, file,
...
         functionality of the code in any way.
-
-        :param max_line_length:   Maximum number of characters for a line.
-                                  When set to 0 allows infinite line length.
-        :param indent_size:       Number of spaces per indentation level.
-        :param pep_ignore:        A list of errors/warnings to ignore.
-        :param pep_select:        A list of errors/warnings to exclusively
-                                  apply.
+
```
[the remark this answers is not in the document]
Comment (AkshJain99) [reply bears/python/PEP8Bear.py]: what change is there in this patch???

File: bears/python/PEP8Bear.py (hunk 1 above)
Comment (apb7) [reply bears/python/PEP8Bear.py]: Please add back the white space between `code.`  and `This`. Change:
`Detects and fixes PEP8 incompliant code.This bear will not change` to `Detects and fixes PEP8 incompliant code. This bear will not change`

File: bears/python/PEP8Bear.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (apb7) [reply bears/python/PEP8Bear.py]: You need to remove the whitespaces which you've added in [this line](https://github.com/coala/coala-bears/pull/2695/files/bf9ac899eba77f590bb75c2c6aff7a89747e49dd#diff-85cbdb9ffc9d72ae7e6a1b21154ee82fR33).

File: bears/python/PEP8Bear.py (hunk 1 above)
Comment (AkshJain99) [reply bears/python/PEP8Bear.py]: okay, 
but  i think it doesn't solve the issue only

**Pull request closed without merging**